packages/varlet-ui/src/icon/Icon.vue

```diff
@@ -7,12 +7,12 @@
         [namespace !== n(), namespace],
         `${namespace}--set`,
         [isURL(name), n('image'), `${namespace}-${nextName}`],
-        [shrinking, n('--shrinking')]
+        [animateInProgress, animationClass || n('--shrinking')]
```

Comment:
change to `animationClass == null ? a : b`

Reply:
why use `animationClass == null ? a : b`? 

use default class when animationClass is undefined, null and empty string, whether or not animateInProgress is on is determined by the transition

Reply:
try to avoid using `truthy` and `falsy`, There are too many possibilities of `falsy` values generated in js, which is not conducive to code reading. 

props should also avoid giving a default value for coding convenience, and should strictly distinguish between `falsy` values and those not passed by the user.

Reply:
We have some old code with the same problem and we should resolve them too

Reply:
I think we should use a function to narrow the null, which might be better

```js
function isEmpty(value){
  return value == null
}

function isEmptyStr(value){
  return isEmpty(value) || value === ''
}
```